diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7a0b4c6..7a20ee9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -32,7 +32,7 @@ pkg_check_modules(GioUnix IMPORTED_TARGET gio-unix-2.0)
 
 option(ENABLE_X11 "Enable X11 support" On)
 option(ENABLE_WAYLAND "Enable wayland support" On)
-option(INATALL_THEME "Install theme files" Off)
+option(INSTALL_THEME "Install theme files" Off)
 
 if (ENABLE_X11)
     set(REQUIRED_XKBCOMMON_COMPONENTS ${REQUIRED_XKBCOMMON_COMPONENTS} X11)
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index d2d8951..518d5ff 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1,4 +1,4 @@
-if (INATALL_THEME)
+if (INSTALL_THEME)
     add_subdirectory(themes)
 endif()
 
